packages/experiments/src/components/Form/inputs/tagPicker/FormTagPicker.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 
 // Components
 import { IFormTagPickerProps } from './FormTagPicker.types';
-import { TagPicker, ITag, ITagPickerProps } from 'office-ui-fabric-react/lib/components/pickers/TagPicker/TagPicker';
+import { TagPicker, ITag, ITagPickerProps } from 'office-ui-fabric-react/lib/Pickers';
```

Comment:
import from `office-ui-fabric-react` instead so consumers with a hard dependency on CommonJS are not blocked...

Reply:
Done.

Reply:
Hey @Vitalius1 after some more discussion with David, we decided to optimize for AMD consumers so.. that means your original import path is preferable. Sorry for the back and forth.